homeassistant/components/sensor/noaa_tides.py

```diff
@@ -0,0 +1,128 @@
+"""
+This component provides HA sensor support for the NOAA Tides and Currents API.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.noaa_tides/
+"""
+import logging
+from datetime import datetime, timedelta
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (ATTR_ATTRIBUTION, CONF_NAME, STATE_UNKNOWN)
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+REQUIREMENTS = ['py_noaa==0.2.2']
+
+_LOGGER = logging.getLogger(__name__)
+
+API_URL = 'https://tidesandcurrents.noaa.gov/api/datagetter'
+CONF_STA_ID = 'station_id'
+CONF_TZ = 'timezone'
+CONF_UNITS = 'units'
+DEFAULT_ATTRIBUTION = "Data provided by NOAA"
+DEFAULT_NAME = 'NOAA Tides'
+DEFAULT_TZ = 'lst_ldt'
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=60)
```

Comment:
Is this the limit for the noaa api?

Please also add a `SCAN_INTERVAL` constant that sets the default scan interval of this platform.

Reply:
They don't specify a limit on their API description page. I figured 1 hour would be a safe estimate. I had removed the `SCAN_INTERVAL` that was previously present and replaced it with this. My mistake.